timeboost-config/src/lib.rs

```diff
@@ -8,10 +8,11 @@ use std::path::{Path, PathBuf};
 pub use chain::{ChainConfig, ChainConfigBuilder};
 pub use committee::{CommitteeConfig, CommitteeDefinition, CommitteeMember, MemberFile};
 pub use contract::CommitteeContract;
+pub use contract::fetch_current;
```

Comment:
Exported from the crate's root, the name could be more specific, e.g. `current_committee`, or `fetch_current_committee`.